app/services/fees/create_from_usage_threshold_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Fees
+  class CreateFromUsageThresholdService < BaseService
+    def initialize(usage_threshold:, invoice:)
+      @usage_threshold = usage_threshold
+      @invoice = invoice
+
+      super
+    end
+
+    def call
+      fee = Fee.new(
+        subscription: invoice.subscriptions.first,
+        invoice:,
+        usage_threshold:,
+        invoice_display_name: usage_threshold.threshold_display_name,
+        invoiceable: usage_threshold,
+        amount_cents: usage_threshold.amount_cents,
```

Comment:
Is this the right amount? 🤔 

Should this not take into account the already invoiced_usage of the lifetime_usage?